resources/model/OpenStudio.idd

```diff
@@ -36395,9 +36395,14 @@ OS:EnergyManagementSystem:Actuator,
   A5, \field Actuated Component Control Type
        \required-field
        \type alpha
-  A6; \field Zone Name
+  A6; \field Zone or Space Name
+       \note This field is useful for a SpaceLoadInstance that is assigned to a SpaceType
+       \note because internally E+ will duplicate the instances and you need to access the final name
+       \note If you are using the Space Translation of the ForwardTranslator (which is defaulted on) this should be set to a Space
+       \note If not then use a ThermalZone
        \type object-list
-       \object-list AllObjects
+       \object-list SpaceNames
+       \object-list ThermalZoneNames
```

Comment:
Constrain this field in the IDD. It will now only accept a Thermal Zone or a Space.